src/features/wallet/send/SendView.tsx

```diff
@@ -334,7 +281,7 @@ const SendView = ({ scanResult, sendType, hotspot, isSeller }: Props) => {
       Alert.alert(t('generic.error'), t('send.error'))
       throw new Error('failed to create seller TransferHotspotV1 transaction')
     }
-    const transfer = createTransfer(
+    const transfer = await createTransfer(
```

Comment:
Noticed this wasn't previously awaiting the promise from `createTransfer`. Not exactly sure what the user-facing impact is here because this function returns undefined anyway (gets past the `!transfer` check because `transfer` is actually a promise).

Reply:
Oh good catch!